pkg/apiserver/logs/controller.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logs
+
+import (
+	"archive/tar"
+	"context"
+	"database/sql"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+)
+
+var controller *Controller
+
+func init() {
+	controller = NewController("./sqlite.db")
```

Comment:
- we may need this sqlite instance across multiple controllers. please move it to global. (maybe consider introducing some standard MVC framework?)
- prefer to put it behind our `config.DataDir`